ts/import-csv/FieldMapper.svelte

```diff
@@ -29,18 +27,17 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 
     $: onNotetypeChange(notetypeId, columnNames);
 
+    // first index is treated specially, because it must be assigned some column
     let firstFieldIndex: number = 0;
+    // other indices are shifted by -1, because the first option is n/a, which
+    // is represented by a negative value
```

Comment:
We could potentially simplify our code a bit here by not special-casing the first field here, and instead showing an error at submission time if the first field is not mapped.

Reply:
Do you mind this complexity? I'd have said that it's worthwhile, if it prevents invalid user input.

Reply:
I don't feel strongly about it, so no need to change if you prefer it this way.